tests/adapter/dbt/tests/adapter/dbt_debug/test_dbt_debug.py

```diff
@@ -51,6 +51,13 @@ def test_ok(self, project):
         run_dbt(["debug"])
         assert "ERROR" not in self.capsys.readouterr().out
 
+    def test_connection_flag(self, project):
+        run_dbt(["debug", "--connection"])
+
+        run_dbt(["debug", "--connection", "--target", "NONE"], expect_pass=False)
+
+        run_dbt(["debug", "--connection", "--profiles-dir", "NONE"], expect_pass=False)
```

Comment:
Do we need the assert from `test_ok` here? i.e.
`assert "ERROR" not in self.capsys.readouterr().out`

Reply:
Oh good point. Adding output checks also indirectly tests a _lot_ of logic changes about how messages are rendered so that's a worthy addition even if not clear ta first glance what's happening.